frontend/src/javascript/utils/axiosUtils.js

```diff
@@ -23,7 +23,13 @@ export function sendRequest(url, data = {}, method) {
             console.error(error);
             let errorMessage = error.message + (error.response ? "\nDetails: " + error.response.data.error : "");
             console.error(errorMessage);
-            toast.error(toastStrings.error ? toastStrings.error : "Wystąpił błąd! Spróbuj ponownie później.");
+            toast.error(
+                url.startsWith("/compiler/compile") && error.response.data
```

Comment:
w tej funkcji do url dodawany jest adres backendu, więc w tym miejscu url to nie jest "/compiler/compile" tylko "http://localhost:8080/compiler/compile". Najprościej to rozwiązać zamieniając startsWith na includes